src/Turbo/tests/app/.gitignore

```
@@ -0,0 +1,2 @@
+/var/
+/public/build/
```

```suggestion
/public/build/
/node_modules/
/yarn.lock
```

Those were ignored and no more with your changes.

But i like the idea of a specific file

---

Just double check Turbo and let's merge this one, i'll prepare a dedicated one for .gitattributes to not you to feel harassed :)

Thanks a lot for taking the time